test/Test/Main.purs

```diff
@@ -151,3 +152,8 @@ testRecordInstances = do
   assert "Record top" $
     (top :: { a :: Boolean }).a
     == top
+
+testSignum :: AlmostEff
+testSignum = do
+  assert "signum positive zero" $ signum 0.0 == 0.0
+  assert "signum negative zero" $ signum (-0.0) == (-0.0)
```

Comment:
This doesn't work unfortunately - positive 0 compares equal to negative zero. Maybe try `show`-ing the result?

Reply:
Changing the test to
```purescript
assert ("signum negative zero: " <> show (signum (-0.0))) $ show (signum (-0.0)) == "-0.0"
```
produces this error:
```
Error: signum negative zero: 0.0
```

Reply:
And `assert (show (-0.0)) $ false` produces `Error: 0.0`

Reply:
Looks like [`Object.is`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is#using_object.is) is a way to verify this.